MIP64/MIP64.md

```diff
@@ -1,4 +1,4 @@
-# MIP64: Bug Bounty Program for MakerDAO Critical Infrastructure
+# Bug Bounty Program for MakerDAO Critical Infrastructure
```

Comment:
This should be left unchanged

Reply:
Are you sure? What about the green highlighted prefix, 
![Screenshot from 2022-01-26 13-51-39](https://user-images.githubusercontent.com/2002771/151253270-322c6a86-0abc-4fb3-9f42-38b27ab3d516.png)

Reply:
Seems like you just caught a bug! :)